recipes/git-annex-remote-rclone/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "git-annex-remote-rclone" %}
+{% set version = "0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DanielDent/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: a119e3a4c194ff24a07aa1620714d60e0dc2a3148f5479cb07faf31e0e31889c
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: mkdir -p $PREFIX/bin && cp git-annex-remote-rclone $PREFIX/bin
```

Comment:
Is this a repackage? Can we build it from source?

Reply:
It's a bash script that we're acquiring directly from the GitHub repository.

Reply:
the entire project is just a bash script - so nothing to build etc. `cp` should be good/enough